docker/service/src/main/java/uk/nhs/adaptors/scr/services/ScrService.java

```diff
@@ -19,7 +20,7 @@
     private static final Mustache REPC_RM150007UK05_TEMPLATE =
         TemplateUtils.loadTemplate("REPC_RM150007UK05.mustache");
 
-    public void handleFhir(Bundle resource) {
+    public void handleFhir(Bundle resource) throws FhirMappingException {
         GpSummary gpSummary = GpSummaryParser.parseFromBundle(resource);
         String spineRequest = TemplateUtils.fillTemplate(REPC_RM150007UK05_TEMPLATE, gpSummary);
 
```

Comment:
What is happening with TODO items below?

Reply:
Will be done when we know "polling" detaills